Formula/zoom.rb

```diff
@@ -1,19 +1,20 @@
 class Zoom < Formula
     desc "Video Conferencing, Web Conferencing, Webinars, Screen Sharing"
     homepage "https://zoom.us"
-    version "5.1.422789.0705"
+    version "5.2.440215.0803"
 
-    url "https://zoom.us/client/#{version}/zoom_x86_64.tar.xz"
-    sha256 "ae1c2faf6b31b114fe0131f0ffabdf007b20983a1123b37cef4bf7a63a2984e9"
+    url "https://zoom.us/client/latest/zoom_x86_64.tar.xz"
```

Comment:
I tried `https://zoom.us/client/5.2.440215.0803/zoom_x86_64.tar.xz` and it is valid, so let's use it.

Putting `latest` in the URL is risky - next time zoom.us updates the software on their end, you will get a hash mismatch when you reinstall the software (b/c `https://zoom.us/client/latest/zoom_x86_64.tar.xz` always points to the latest version w/o telling you the exact version number).

Reply:
Ok, resolve it please

Reply:
Please replace `latest` with `#{version}` on your end, I don't have write permission to your branch.